lib/Server/ReadStream.js

```diff
@@ -29,10 +29,11 @@ ReadStream.prototype._transform = function (chunk, encoding, next) {
 		this.pending = Buffer.concat([ this.pending, chunk ]);
 	}
 
-	while (this.pending.length >= 6) {
```

Comment:
This was correct, as we only need 6 bytes to read package length (4 offset + 2 bytes of the length). I know the package should be larger but we can't assume that.